tox.ini

```diff
@@ -17,7 +17,7 @@ deps =
     django14: Django>=1.4,<1.5
     django15: Django>=1.5,<1.6
     django16: Django>=1.6,<1.7
-    django17: Django>=1.7,<1.8
+    django17: Django>=1.7,<1.8a
```

Comment:
Are you sure that is needed? By default pip ignores "pre" releases, so "<1.8" should be fine.